src/integrationtest/python/GeoAttribute_tests.py

```diff
@@ -0,0 +1,48 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+from janusgraph_python.core.attribute.Geo import Geo
+from janusgraph_python.core.datatypes.GeoShape import GeoShape
+
+
+class TestGeoAttributes(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        docker_ip = Popen(["docker-machine", "ip"], stdout=PIPE).communicate()[0]
+        docker_ip = docker_ip.strip().decode("utf-8")
+
+        self.client = JanusGraphClient()
+        self.client = self.client.connect(host=str(docker_ip), port="8182",
+                                          traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
+        self.client.close()
+        self.container.stop()
+
+    def test_geo_contains(self):
+
+        # First we will need to add a property so that GeoContains can be queried.
+        place = GeoShape.Circle(50, 100, 100)
+        self.g.V().has("name", "tartarus").property("place", place).next()
+
+        truth = {GeoShape.Circle(50, 100, 10): 1, GeoShape.Circle(50, 100, 500): 0}
```

Comment:
This seems suspect to me. There are 2 GeoShape circles with the same latitude and longitude, but the one with a smaller radius is expected to return a higher count? I definitely feel like I need to see these running in CI.

Reply:
I tested that on my Console. but messaged you on Slack regarding checking that out. But the logic why this should hold true sounds good here.

Just in previous line, we add a `geoshape circle` with `radius 100`. In successive line, we test against 2 `geoshapes` namely:

- Query 1: Which geometrical objects in graph contains the `circle` object with `radius 10`. It should return 1 (count) the one we just added before.
- Query 2: Which geometrical objects in graph contains the `circle` object with `radius 500`. It should return 0 (count) . As the object we added was of radius 100, hence this query fails.